extensions/amp-timeago/0.2/test/test-amp-timeago.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import '../amp-timeago';
+import {act} from 'react-dom/test-utils';
```

Comment:
I don't think React's tools will work for Preact. We want https://preactjs.com/guide/v10/unit-testing-with-enzyme/#triggering-state-updates-and-effects-with-act.

Reply:
Good catch, though the failing test doesn't make it as far as using this 😅  Looks like it is an issue with retrieving the IntersectionObserver callback during instantiation since it fails with `observerCallback is not a function` 

Removing the test case for now. This may be a better direct test on the Component anyway rather than in the AMP layer.